src/data/finsupp/basic.lean

```diff
@@ -1152,6 +1152,14 @@ lemma finset_sum_apply [add_comm_monoid N] (S : finset ι) (a : α) (f : ι →
   (f.sum g) a₂ = f.sum (λa₁ b, g a₁ b a₂) :=
 finset_sum_apply _ _ _
 
+lemma coe_finset_sum [add_comm_monoid N] (S : finset ι) (f : ι → α →₀ N) :
+  ⇑(∑ i in S, f i) = ∑ i in S, f i :=
+(coe_fn_add_monoid_hom : (α →₀ N) →+ _).map_sum _ _
+
+lemma coe_finsupp_sum [has_zero M] [add_comm_monoid N] (f : α →₀ M) (g : α → M →₀ N) :
+  ⇑(∑ i in f.support, g i) = ∑ i in f.support, g i :=
+coe_finset_sum _ _
```

Comment:
The whole point of this lemma is to be stated about `finsupp.sum`!
```suggestion
lemma coe_sum [has_zero M] [add_comm_monoid N] (f : α →₀ M) (g : α → M → β →₀ N) :
  ⇑(f.sum g) = f.sum (λ a₁ b, g a₁ b) :=
coe_finset_sum _ _
```
(note I removed `finsupp` from the name to match `sum_apply`.

Reply:
Sorry, not making  good day of it today!